fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -643,6 +691,12 @@ public void replaceProperties(final IdentifierConverter<Resource, FedoraResource
             }
 
             removeEmptyFragments();
+            try {
+                touch();
```

Comment:
In a similar fashion to `updateProperties()`, it seems like we should be doing some sort of `touchLdpMembershipResource` here in `replaceProperties()` as well, no?